src/database/postgres/hash.js

```diff
@@ -46,7 +50,11 @@ module.exports = function (db, module) {
 				async.apply(helpers.ensureLegacyObjectType, tx.client, key, 'hash'),
 				async.apply(query, {
 					name: 'setObjectField',
-					text: 'INSERT INTO "legacy_hash" ("_key", "data") VALUES ($1::TEXT, jsonb_build_object($2::TEXT, $3::TEXT::JSONB)) ON CONFLICT ("_key") DO UPDATE SET "data" = jsonb_set("legacy_hash"."data", ARRAY[$2::TEXT], $3::TEXT::JSONB)',
+					text: `
+INSERT INTO "legacy_hash" ("_key", "data")
```

Comment:
Any reason these aren't indented evenly? I didn't think SQL had significant whitespace.

Reply:
I'm using the same format I use for SQL at work, where the first keyword per line is right-aligned with other things at the same nesting level.